source/configuration_manual/mail_crypt_plugin.rst

```diff
@@ -326,7 +326,11 @@ Hence, this is possible:
   }
 
   passdb static {
-    args = password=pass crypt_global_public_key=<content of ecpubkey.pem> crypt_global_private_key/main/private_key=<content of ecprivkey.pem>
+    static_password = pass
```

Comment:
This can actually be simply "password = pass" since it auto-prefixes passdb_static_. Same everywhere else.